### PR TITLE
Operations: Add tooling/docs/monitoring for thanos bucket inspect/replicate

### PR DESCRIPTION
**Pull request by philipgough**
https://issues.redhat.com/browse/RHOBS-816

Screenshot 2023-08-02 at 11 56 20

### CODE REVIEW

File: resources/operations/bucket-inspect/job-template.yaml
Comment (saswatamcode) [inline resources/operations/bucket-inspect/job-template.yaml:1]: This is the one-off inspect?

File: resources/operations/bucket-replicate/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# What
+
+This template deploys [Thanos Bucket Inspect](https://thanos.io/tip/components/tools.md/#bucket-insepct)
+as a Kubernetes Job.
+
+# SOP
+
+> **_NOTE:_**  Before running this Job, if you wish to track progress via logs,
```
Comment (saswatamcode) [inline resources/operations/bucket-replicate/README.md:8]: I'd also add a note here, that depending on number of blocks in the `from` bucket...this job might be over even before single Prometheus scrape.

**Review by saswatamcode — APPROVED** [review]
Thanks! Generally LGTM!

File: resources/operations/bucket-inspect/job-template.yaml
Comment (philipgough) [reply resources/operations/bucket-inspect/job-template.yaml:1]: yes one of inspect. would suggest we run it on the original source and then compare iteratively against the cronjob. Also very useful for testing your s3 config and secrets

File: resources/operations/bucket-replicate/README.md (hunk 1 above)
Comment (philipgough) [reply resources/operations/bucket-replicate/README.md:8]: yeah, good idea

**Pull request merged**